package.json

```diff
@@ -35,30 +35,30 @@
     "url": "http://www.bentley.com"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "~7.0.1",
-    "@typescript-eslint/parser": "~7.0.1",
-    "eslint-plugin-deprecation": "^2.0.0",
-    "eslint-plugin-import": "^2.29.1",
+    "@typescript-eslint/eslint-plugin": "~8.8.0",
+    "@typescript-eslint/parser": "~8.8.0",
+    "eslint-formatter-visualstudio": "^8.40.0",
+    "eslint-plugin-import": "^2.30.0",
     "eslint-plugin-jam3": "^0.2.3",
```

Comment:
wonder if we should remove this, hasnt been touched in 5 yrs

Reply:
we have an issue open for that (https://github.com/iTwin/eslint-plugin/issues/74), but I think we should consider that out of scope for this PR